### PR TITLE
Gene page QA

### PR DESCRIPTION
**Pull request by broskoski**
Re: https://github.com/artsy/force/issues/1519

### CODE REVIEW

File: src/components/artwork/fillwidth.tsx [hunk 1]
```diff
@@ -127,8 +126,13 @@ const ArtworkFragment = Relay.QL`
     ${Artwork.getFragment("artwork")}
   }
 `
+const sizeMeOptions = {
```
Comment (mzikherman) [inline src/components/artwork/fillwidth.tsx]: I guess I missed this, what's `sizeMe`?

File: src/components/artwork_grid.tsx [hunk 2]
```diff
@@ -14,17 +14,23 @@ interface Props extends RelayProps, React.HTMLProps<ArtworkGrid> {
 
 interface State {
   loading: boolean,
+  interval: any,
```
Comment (mzikherman) [inline src/components/artwork_grid.tsx]: Nice.

File: src/components/artwork/fillwidth.tsx (hunk 1 above)
Comment (broskoski) [inline src/components/artwork/fillwidth.tsx]: Listens to changes in size for the wrapped component and passes the dimensions down as props.

File: src/components/gene/contents.tsx [hunk 3]
```diff
@@ -136,7 +139,8 @@ export class GeneContents extends React.Component<Props, State> {
   }
 
   loadMoreArtworks() {
-    if (!this.state.loading) {
+    const hasMore = this.props.gene.filtered_artworks.artworks.pageInfo.hasNextPage
```
Comment (mzikherman) [inline src/components/gene/contents.tsx]: pssh who goes beyond page 1

Comment (broskoski) [inline src/components/gene/contents.tsx]: Not me

**Pull request merged**